### ISSUE TITLE
[studio] Stack trace log  when starting up.

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#4257]
## Describe the bug
There is stack trace log during start up.

## To Reproduce
1. Startup Authoring. 
2. See stack trace in Tomcat log

## Expected behavior
No stack trace log during start up. 

## Screenshots
N/A

## Logs
```
[INFO] 2020-08-24T10:10:57,960 [localhost-startStop-1] [http.DefaultFilterChainValidator] | Checking whether login URL '/login' is accessible with your configuration 
[INFO] 2020-08-24T10:10:57,969 [localhost-startStop-1] [http.DefaultFilterChainValidator] | Unable to check access to the login page to determine if anonymous access is allowed. This might be an error, but can happen under normal circumstances. 
java.lang.UnsupportedOperationException: public abstract java.lang.String javax.servlet.ServletRequest.getParameter(java.lang.String) is not supported
	at org.springframework.security.web.UnsupportedOperationExceptionInvocationHandler.invoke(FilterInvocation.java:235) ~[spring-security-web-5.3.3.RELEASE.jar:5.3.3.RELEASE]
	at com.sun.proxy.$Proxy154.getParameter(Unknown Source) ~[?:?]
	at javax.servlet.ServletRequestWrapper.getParameter(ServletRequestWrapper.java:153) ~[servlet-api.jar:3.1.FR]
	at org.craftercms.studio.impl.v1.web.security.access.StudioSiteAPIAccessDecisionVoter.vote(StudioSiteAPIAccessDecisionVoter.java:59) ~[classes/:3.2.0-SNAPSHOT]
	at org.craftercms.studio.impl.v1.web.security.access.StudioAccessDecisionManager.decide(StudioAccessDecisionManager.java:43) ~[classes/:3.2.0-SNAPSHOT]
	at org.springframework.security.config.http.DefaultFilterChainValidator.checkLoginPageIsntProtected(DefaultFilterChainValidator.java:206) ~[spring-security-config-5.3.3.RELEASE.jar:5.3.3.RELEASE]
	at org.springframework.security.config.http.DefaultFilterChainValidator.validate(DefaultFilterChainValidator.java:51) ~[spring-security-config-5.3.3.RELEASE.jar:5.3.3.RELEASE]
	at org.springframework.security.web.FilterChainProxy.afterPropertiesSet(FilterChainProxy.java:168) ~[spring-security-web-5.3.3.RELEASE.jar:5.3.3.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1855) ~[spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1792) ~[spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:595) ~[spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517) ~[spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323) ~[spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:226) [spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321) [spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202) [spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:893) [spring-beans-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:879) [spring-context-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:551) [spring-context-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:401) [spring-web-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:292) [spring-web-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:103) [spring-web-5.2.7.RELEASE.jar:5.2.7.RELEASE]
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4689) [catalina.jar:8.5.57]
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5155) [catalina.jar:8.5.57]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:8.5.57]
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743) [catalina.jar:8.5.57]
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719) [catalina.jar:8.5.57]
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705) [catalina.jar:8.5.57]
	at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:970) [catalina.jar:8.5.57]
	at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1840) [catalina.jar:8.5.57]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

### Version
`3.1.9` & `3.2.x`

### OS
Any

### Browser
N/A

## Additional context
N/A

**Comment by jdrossl** [comment]
For 3.1.x it will be handled via logging configuration, for 3.2.x it will be fixed as part of https://github.com/craftercms/craftercms/issues/4361

**Comment by yacdaniel** [comment]
Verified fixed in `3.1.12`

**Comment by yacdaniel** [comment]
Verified fixed in `4.0.0`
